src/manage_sql.c

```diff
@@ -53337,12 +53345,18 @@ manage_set_radius_info (int enabled, gchar *host, gchar *key)
       g_free (quoted);
     }
 
-  if (key)
+  if (key && strcmp (key, "ThisIsAPlaceholderSecretKey!"))
```

Comment:
I don't like checking for a placeholder string here. I think this should be handled by GSA and gsad only sending the key if it was changed, so this can be left as it was or maybe changed to check for an empty string.